dwst/scripts/controller/test/error.js

```diff
@@ -64,7 +64,7 @@ describe('ErrorHandler class', () => {
     });
     it('should mlog InvalidParticles error', () => {
       errorHandler.onDwstError(new InvalidParticles(...[
-        [')'],
+        ['")"'],
```

Comment:
Maybe the test should have two or three items so the tests includes the *or* functionality.